openreview/conference/builder.py

```diff
@@ -2109,7 +2110,7 @@ def get_result(self):
 
         venues = self.client.get_group(id = 'venues', details='writable')
         if venues.details.get('writable'):
-            self.client.add_members_to_group('venues', home_group.id)
+            self.client_v2.add_members_to_group('venues', home_group.id)
```

Comment:
This is code is part of the Builder class not the conference so you can create the client_v2 in the Builder class too.

Reply:
some tests are failing because of this: https://github.com/openreview/openreview-py/pull/1934/files#diff-8d0f7e15bc90c791f490e1aef09fa75304f454f2e25fb186568515716e68c2b7L1844